changelog/v0.26.2/support-k8s-1.25.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+    - type: NON_USER_FACING
```

Comment:
is this no longer a breaking change?

Reply:
not since we merged in the breaking changes that became v.0.26, I think?

Reply:
nah its breaking

Reply:
i think we are going to have issues with gloo edge LTS branches picking up future skv2 patch fixes if this isn't marked as a breaking change. mostly concerned about the go mod updates to 0.25 k8s api